pyproject.toml

```diff
@@ -1,3 +1,31 @@
+[project]
+name = "s2geometry"
+description = "Computational geometry and spatial indexing on the sphere"
+authors = [
+    { name = "Brian Miles", email = "selimnairb@gmail.com" },
```

Comment:
Is this supposed to be the author of the project or the author of the `pyproject.toml` file?

Reply:
this should be the project author (you can include multiple)

Reply:
@jmr Agreed, it shouldn't be me.